ci/run_premerge_gpu_tf.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
```

Comment:
may I ask why we dup most of code/script here? 

it would be nice to have an arg for func `integration_test()` at `ci/run_premerge_gpu.sh`
```
...
./run_integration_tests.sh -m ${1}
...
```

simply call the func
```
integration_test tensorflow
integration_test pytorch
```
when needed.

Could also extend the `BUILD_TYPE` args part to control which test/tests should be run

Reply:
Thanks for the feedback, let me modify this.